src/main/java/frc/robot/Constants.java

```diff
@@ -0,0 +1,27 @@
+/* ==================================================
+Authour: Shane Pinto & Lucas Jacobs
+Description: Constants.java contains most of the 
+constant values used throughout the robot. This 
+includes settings, hardware ports, and other values.
+================================================== */
+
+package frc.robot;
+
+public final class Constants
+{
+    // HARDWARE PORTS
+   
+
+    // CONTROLLER COMPUTER PORTS
+    public static final int DUALSHOCK = 0;
+
+    // LIMIT SWITCH PORTS
+
+
+    // SETTINGS
+    public static final double DEADZONE = 0.15; // Deadzone applied to joysticks to aid in adjusting sensitivity
```

Comment:
Is this deadzone applied directly to the raw controller values and not the speed values(like from `getY()` and `getX()`)?  If not, and it is meant to be applied on the actual motor speed, I would be wondering why the deadzone is half of the `MAX_THROTTLE_SPEED`, since that seems pretty large of a deadzone.

Reply:
You're right, it is a pretty large deadzone. I've changed the code so that it is applied on the raw controller values.